### PR TITLE
HTTP/2 check on BBC Homepage

### PR DESCRIPTION
**Pull request by aidylewis**
k6 supports HTTP/2

### CODE REVIEW

File: samples/http_2.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import http from "k6/http";
+import { check } from 'k6';
```
Comment (liclac) [inline samples/http_2.js]: Please use double quotes: `"k6"`.

File: samples/http_2.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+export default function() {
+  const HOMEPAGE_RESPONSE = http.get("https://www.bbc.co.uk/");
+  check(HOMEPAGE_RESPONSE, { "HTTP 2.0": (r) => r.proto  === "HTTP/2.0" });
```
Comment (liclac) [inline samples/http_2.js]: The formatting used in the rest of the samples is:

```js
check(http.get("https://www.bbc.co.uk/"), {
    "status is 200": (r) => r.status == 200,
    "protocol is HTTP/2": (r) => r.proto == "HTTP/2.0",
});
```

Comment (liclac) [reply samples/http_2.js]: Also: are you sure this is correct? I seem to remember the `proto` value for HTTP/2 being `"h2"`, but I could be wrong.

Comment (aidylewis) [reply samples/http_2.js]: https://github.com/loadimpact/k6/blob/80c201125846f9954312e8f65cc40111b04c5fc4/js/modules/k6/http/http_test.go#L219

**Comment by aidylewis** [comment]
I did a **reset --hard** and **push force** (Probably not the best way). 

Why checks with value equality allowing coercion and not without coercion?

**Comment by aidylewis** [comment]
Probably ```proto``` should be `h2`. WDYT?

**Comment by liclac** [comment]
Test run the script and see which one passes >___>

And I honestly don't give a damn which one it is, it's a bit inconsistent as it is.

File: samples/http_2.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import http from "k6/http";
+import { check } from "k6";
+
+export default function() {
+  const HOMEPAGE_RESPONSE = http.get("https://www.bbc.co.uk/");
```
Comment (liclac) [inline samples/http_2.js]: Now you're just making the request twice.

Comment (aidylewis) [reply samples/http_2.js]: Fails with 'h2'.

Comment (aidylewis) [reply samples/http_2.js]: Sorry. I will amend on squash.

**Comment by aidylewis** [comment]
Created a branch, pushed upstream and then created a PR. Which I think is the correct process. Thanks @robingustafsson

**Pull request closed without merging**